packages/forma-36-react-components/src/components/Icon/svg/Puzzle.tsx

```diff
@@ -2,9 +2,8 @@ import * as React from 'react';
367583 2.05932911l.007524.21236042-.0015 3.59982283.7615.0001c.8494108 0 1.6694205.2855341 2.331661.803425l.176603.1469639.168201.1582461c.7098106.7098105 1.108535 1.6724867 1.108535 2.676365 0 1.0038989-.398739 1.9666691-1.108535 2.676465-.6551962.6551963-1.5259445 1.04535-2.4456225 1.1014967l-.2308425.0070383-.7625-.0001-.0000008 3.6021972c-.001509.7097686-.2371401 1.3949403-.6640517 1.9523524l-.1337691.1632877-.1449888.1550031c-.5517704.5522891-1.2834668.8837199-2.0585347.9371294l-.2123549.0076302h-5.2625l.0001868-2.6356662c-.0174258-.5519281-.2489804-1.0754897-.6456342-1.4598172-.3965955-.3842709-.9271914-.5991166-1.4795526-.5991166-.5523557 0-1.08293313.2148418-1.4795326.5991166-.35257236.3416159-.5747083.7932537-.63185061 1.2687305l-.01362679.1792529v2.6475h-5.26064719c-.85246365-.0014718-1.6695948-.3406448-2.27250537-.9432725-.55265314-.5523928-.88405125-1.2849876-.93688719-2.0604716l-.00746025-.2124559v-5.2613l2.63566732.0001867c.5519788-.0174257 1.07552769-.2489737 1.45983026-.6456281.38429728-.3966403.59918242-.927256.59918242-1.4795586 0-.5522138-.21489294-1.0828264-.5991853-1.4794615-.34158876-.3525678-.79322073-.5747154-1.26872843-.6318961l-.17926627-.0136424h-2.635l.00000033-5.25947326c-.00112764-.85127352.33567029-1.66819322.93641898-2.2713251.55068254-.55287231 1.28187998-.88466793 2.05638222-.9378642l.21219847-.00755744 3.60221-.00088.00029-.76037c0-.84911712.28542096-1.66881527.80313019-2.33082675l.14691272-.17654264.15819163-.16814524c.70958241-.70958463 1.67199375-1.10823237 2.67547546-1.10823237zm-.0012 1.884997c-.5032415 0-.98589124.19991968-1.34174496.55576504-.3163123.3163123-.50941462.73280033-.54842115 1.17489485l-.00734389.16684011v2.6475h-5.48875c-.68072208 0-1.24414788.52433021-1.31191703 1.19419815l-.00683297.13580185-.00079 3.37412.75079.0009c.98820242 0 1.93878497.3644982 2.6719907 1.0190548l.16526074.1561666c.75248132.7524041 1.17524856 1.773083 1.17524847 2.8375353-.00072802 1.0636788-.42377298 2.0834966-1.17621016 2.8355036-.69867083.6981689-1.62822477 1.1122206-2.60910371 1.1678088l-.22718604.0064309-.76079-.0004.00079 3.3754c0 .6869835.53182749 1.2589531 1.20268154 1.3238665l.12731846.0061335 3.37421-.0004.00079009-.7623567c.00067603-.9875212.36534828-1.9372872 1.01963952-2.6699341l.15609759-.1651365c.75215585-.7518028 1.77199216-1.1743418 2.8355895-1.1746727 1.0637192.0003309 2.0837533.4230399 2.8359483 1.175235.6984206.6984205 1.1128294 1.6278448 1.1687323 2.6087661l.0065027.2271989-.0003.7609 3.374-.0009c.6836073 0 1.2533415-.5269875 1.321888-1.1936252l.006912-.1350748v-5.4875h2.6487c.5034404 0 .9862587-.1999887 1.342235-.555965.3559684-.3559685.555965-.8387836.555965-1.342135 0-.5034404-.1999887-.9862587-.555965-1.342235-.3164234-.3164233-.7330676-.5095973-1.175331-.5486184l-.166904-.0073466h-2.6487v-5.48747c0-.6875945-.5312955-1.26013223-1.2015872-1.32511038l-.1272128-.00613962h-5.4887v-2.6475c0-.50326472-.1999057-.98589232-.5557557-1.34172562-.3558859-.35586089-.8385119-.55577438-1.3417443-.55577438z" />
```

Comment:
Is it optimized with SVGO?

Reply:
Please, update the icon also in `forma-v4` branch.

Reply:
I would ask that to @fabe as I wasn't creating this svg

Reply:
Looks unoptimised

Reply:
I'll replace it with the optimised version in a minute